bin/web/src/lib/analysis.svelte.ts

```diff
@@ -42,7 +42,7 @@ export type PacketAnalysis = {
 export type Event = QualitativeWarning | InformationalEvent;
 export enum EventType {
     Informational,
-    Warning,
+    Warning
```

Comment:
nit: i do really like having trailing commas for items like this, is there a linter setting you could enable to preserve them?